### PR TITLE
Add alt attribute for thread mailer images

### PR DESCRIPTION
**Pull request by HSalmon**
Addresses: https://github.com/loomio/loomio/issues/3153

### CODE REVIEW

**Comment by gdpelican** [comment]
I see a couple more thread mailer images, should these have alt tags too? 🔴 

screen shot 2016-04-27 at 2 37 56 pm

File: app/views/thread_mailer/_new_comment.html.haml [hunk 1]
```diff
@@ -9,7 +9,7 @@
             - if user.avatar_kind == 'initials'
               = user.avatar_initials
             - else
-              = image_tag user.avatar_url(:medium)
+              = image_tag user.avatar_url(:medium), alt: "user avatar"
```
Comment (gdpelican) [inline app/views/thread_mailer/_new_comment.html.haml]: Can this be the user's name or username? 💛

**Comment by HSalmon** [comment]
Done!

**Comment by gdpelican** [comment]
Cool! 🍏

**Pull request merged**